--- lksctp-tools-1.0.7/src/apps/Makefile.am.orig	2005-10-21 21:24:20.000000000 +0200
+++ lksctp-tools-1.0.7/src/apps/Makefile.am	2007-08-12 09:49:41.129764470 +0200
@@ -8,8 +8,8 @@
 INCLUDES = -I. -I$(top_srcdir)/src/include -I$(top_srcdir)/src/testlib 
 AM_CFLAGS = -g -Wall -Wstrict-prototypes -Wimplicit-function-declaration
 AM_LDFLAGS = 
-LDADD = $(top_builddir)/src/lib/libsctp.la \
-	$(top_builddir)/src/testlib/libsctputil.la
+LDADD = $(top_builddir)/src/testlib/libsctputil.la \
+	$(top_builddir)/src/lib/libsctp.la
 
 # programs to be installed with the distriubution
 bin_PROGRAMS = sctp_darn sctp_test
--- lksctp-tools-1.0.7/src/func_tests/Makefile.am.orig	2006-01-24 16:58:52.000000000 +0100
+++ lksctp-tools-1.0.7/src/func_tests/Makefile.am	2007-08-12 10:00:47.931763351 +0200
@@ -8,8 +8,8 @@
 INCLUDES = -I. -I$(top_srcdir)/src/include -I$(top_srcdir)/src/testlib 
 AM_CFLAGS = -g -Wall -Wstrict-prototypes -Wimplicit-function-declaration
 AM_LDFLAGS = -lpthread
-LDADD = $(top_builddir)/src/lib/libsctp.la \
-	$(top_builddir)/src/testlib/libsctputil.la
+LDADD =	$(top_builddir)/src/testlib/libsctputil.la \
+	$(top_builddir)/src/lib/libsctp.la
 
 V6FLAGS = -DCONFIG_IPV6=1 -DTEST_V6=1 ${DEFS} ${INCLUDES} ${CFLAGS}
 
